### DISCUSSION TITLE
TypeError: undefined is not a function (near '...(0,zustand__WEBPACK_IMPORTED_MODULE_1__.create)...')

### DISCUSSION THREAD

**Discussion by egalvan10** [pmndrs/zustand#2639] [Q&A]
I tried to add the demo code to my barebones react app and got the following error:

```TypeError: undefined is not a function (near '...(0,zustand__WEBPACK_IMPORTED_MODULE_1__.create)...')```

This is the complete code:

src/index.js
```
import React from "react";
import { createRoot } from "react-dom/client";
import App from "./app";

const rootElement = document.getElementById("root");
const root = createRoot(rootElement);
root.render(<App />);
```

src/app/index.js
```
import React from "react";
import { create } from "zustand";

//Basic example from Zustand readme
const useBearStore = create((set) => ({
  bears: 0,
  increasePopulation: () => set((state) => ({ bears: state.bears + 1 })),
  removeAllBears: () => set({ bears: 0 }),
}));

function App() {
  return (
    <h1>Hello</h1>
  );
}
export default App;
```

When I build and run this using Webpack I get the mentioned error and a blank page.

react v18.3.1
webpack v5.92.1
zustand v4.5.4

**Comment by dbritto-dev** [comment]
@egalvan10 CRA?

**Reply by egalvan10** [reply]
This is a simplified version of a larger existing app. I essentially commented out the real `App()` module and tried running the zustand code as shown to avoid running the rest of the big app. I am unsure how the app was started (whether as a CRA or otherwise...).

**Reply by dbritto-dev** [reply]
Well, it would be difficult to know what happen is we don't know how you are running/building the app
